elementtypes/TranslateElementType.php

```diff
@@ -241,6 +241,11 @@ public function getSources($context = null)
      */
     public function getIndexHtml($criteria, $disabledElementIds, $viewState, $sourceKey, $context, $includeContainer, $showCheckboxes)
     {
+        // If the site only has 1 locale enabled, set the translated locale to the primary (and only) locale
+        if(empty($criteria['locale'])) {
```

Comment:
Can you place a space between `if` and `(`? Like so: `if (` - this is PSR-2 compliancy.